components/sendgrid/actions/get-a-block/get-a-block.js

```diff
@@ -0,0 +1,34 @@
+const validate = require("validate.js");
+const common = require("../common");
+
+module.exports = {
+  ...common,
+  key: "sendgrid-get-a-block",
+  name: "Get a Block",
+  description: "Gets a specific block.",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    ...common.props,
+    email: {
+      type: "string",
+      label: "Email",
+      description: "The email address of the specific block.",
+    },
+  },
+  methods: {
+    ...common.methods,
+  },
+  async run() {
```

Comment:
Include the `$` variable.
```suggestion
  async run({ $ }) {
```